File: March/CPP/Maximum Odd Binary Number/Maximum Odd Binary Number.cpp
class Solution {
public:
    string maximumOddBinaryNumber(string st) {
        string s = st;
       sort(s.begin(),s.end(),greater<int>());
       for(int i=0;i<st.size()-1;i++){
           s[i]=s[i+1];
       }
       s[st.size()-1]='1';
       return s;
    }
};

File: March/Python/Maximum Odd Binary Number/Maximum Odd Binary Number.py
class Solution:
    def maximumOddBinaryNumber(self, s: str) -> str:
        s = [c for c in s]
        left = 0

        for i in range(len(s)):
            if s[i] == "1":
                s[i], s[left] = s[left], s[i]
                left += 1
        s[left - 1], s[len(s) - 1] = s[len(s) - 1], s[left -1]
        return "".join(s)
